frontend/src/store/index.js

```diff
@@ -164,7 +164,7 @@ const state = {
       description: 'Indicates whether Gardener is able to hibernate this cluster. If you do not resolve this issue your hibernation schedule may not have any effect.'
     }
   },
-  prefersColorScheme: 'light'
+  prefersColorScheme: window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light'
```

Comment:
```suggestion
  prefersColorScheme: 'light'
```
We don't want to have a media query dependency here. This is why we have the mediaPlugin.